src/renderer/components/Execute.tsx

```diff
@@ -32,23 +50,27 @@ const Execute = () => {
         </div>
         <div>
           <h6>Selected Procedure</h6>
-          <ProcedureDropdown
-            activeDb={activeDb}
-            activeProcedure={activeProcedure}
-            setActiveProcedure={setActiveProcedure}
-            setCode={setCode}
-          />
-        </div>
-        <div className="code-wrapper">
-          <p className="procedure-code">{code}</p>
+          <div className="d-flex flex-direction-column">
+            <ProcedureDropdown
+              activeDb={activeDb}
+              activeProcedure={activeProcedure}
+              setActiveProcedure={setActiveProcedure}
+              setCode={setCode}
+            />
+            <Button variant="link" size="sm" onClick={() => setAlert(!!code)}>
+              code
+            </Button>
+          </div>
         </div>
+        <div> </div>
```

Comment:
Is this used for something? if so <div /> does the same thing, not a problem though.

Reply:
That was actually left in by mistake